src/Locations.js

```diff
@@ -202,55 +206,71 @@ export default class Locations extends Component {
-                                            maxHeight: '75vh',
-                                            overflowY: 'auto'
-                                        }}>
-                                        {locationList.map(location => (
-                                            <DroppableLocation
-                                                key={location}
-                                                droppableId={location}
-                                                name={location.split('-').join(' ')}
-                                                style={{flex: 1}}
-                                                isDropDisabled={this.state[location].length > 0}>
-                                                {this.state[location].map((item, index) => (
-                                                    <Draggable
-                                                        key={item}
-                                                        draggableId={item}
-                                                        index={index}>
-                                                        {(provided, snapshot) => (
-                                                            <div
-                                                                ref={provided.innerRef}
-                                                                {...provided.draggableProps}
-                                                                {...provided.dragHandleProps}
-                                                                style={getItemStyle(
-                                                                    snapshot.isDragging,
-                                                                    provided.draggableProps.style
-                                                                )}>
-                                                                {item.split('-').join(' ')}
-                                                            </div>
-                                                        )}    
-                                                    </Draggable>
-                                                ))}
-                                            </DroppableLocation>
-                                        ))}
-                                        </div>
+                                      <h1>{locationGroup}</h1>
+                                      <Tabs>
+                                        <TabList>
+                                          {Object.keys(LOCATIONS[locationGroup]).map((locationName) =>(
+                                            <Tab key={locationName.split(' ').join('-')}>{locationName}</Tab>
+                                          ))}
+                                        </TabList>
+                                          {Object.entries(LOCATIONS[locationGroup]).map((locationName) => (
```

Comment:
You can use argument destructuring here to make the code cleaner: `([locationName, location]) => ...`

Reply:
Learning something new. Thank you very much